CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - Complete architecture with foundation release
-- Multi-backend support: OpenAI (Responses API), Claude (Messages API), Grok (Chat API)
+- Multi-backend support: OpenAI (Responses API), Claude (Messages API), Grok (Chat API), Gemini (Chat API)
```

Comment:
Minor: Shall we keep it consistent to use alphabetic order?

Reply:
Thanks for suggestion!
Done.
<img width="612" height="88" alt="Screenshot 2025-08-05 at 3 48 52 PM" src="https://github.com/user-attachments/assets/4e313526-e4a9-4ee7-b3f9-0b2e995a334a" />